frontend/tests/tests.test.js

```diff
@@ -64,20 +72,34 @@ for (const envName in deploymentConfigs) {
     // Load cset
     // await page.getByRole('link', { name: 'Load concept sets' }).click();  // fail
     await page.getByTestId('load-concept-sets').click();
-    
+
+
     // Select a related cset
     // TODO: problem: id "row-0" is shared by both (i) the first table on the page (selected csets), and (ii) the 2nd table (related csets)
     //  - solution: (a) set different IDs, (b) probably better, data-testid
     // todo: anything better to wait for than row-0?
-    const firstRow = await page.waitForSelector('#row-0');
+
+    // const firstRow = await page.waitForSelector('#row-0');
+    const firstRow = await page.waitForSelector('#related-csets-table #row-0');
```

Comment:
@Sigfried Hmm interesting, I thought it was going to look something like `#related-csets-table > #row-0`. Is / has the `>` always been optional / alternative when selecting child elements?

Reply:
`>` means a direct child element, space means any descendant (i think)